pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -263,17 +287,16 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(pod *v1.Pod) {
 
 	allVolumesAdded := true
 	mountsMap, devicesMap := dswp.makeVolumeMap(pod.Spec.Containers)
+	errEventTemplate := "Error processing volume %q for pod %q: %v"
 
 	// Process volume spec for each volume defined in pod
 	for _, podVolume := range pod.Spec.Volumes {
 		volumeSpec, volumeGidValue, err :=
 			dswp.createVolumeSpec(podVolume, pod.Name, pod.Namespace, mountsMap, devicesMap)
 		if err != nil {
-			glog.Errorf(
-				"Error processing volume %q for pod %q: %v",
-				podVolume.Name,
-				format.Pod(pod),
-				err)
+			msg := fmt.Sprintf(errEventTemplate, podVolume.Name, format.Pod(pod), err)
+			glog.Errorf(msg)
+			dswp.recordFailedProcessPodEvent(pod, msg)
```

Comment:
Currently, the error message is logged (`glog.Errorf(msg)`) into a log file several times per second.
Is it possible to move the `glog.Errorf(msg)` into `dswp.recordFailedProcessPodEvent()` func so that the error message is logged into a log file only once in 10 seconds?

Reply:
Yeap, that can be an option. But still, like I commented in the ticket, the logging message should align with the trace of program execution, so from the log I should see how the program is running. 
@msau42  and @jsafrane, would you please share your thoughts on this?

Reply:
I think as long as it logs the first time it's fine.  It's good to not spam the logs too.

Reply:
Yes, please log at lower frequency, 100ms spams log quite a lot.

Reply:
Ok, let me change the log frequency as well, will follow the same interval as the pod events.